contract/src/pegasus.js

```diff
@@ -288,10 +216,7 @@ const makeCourier = ({
  * @param {BoardDepositFacet} board where to find depositFacets
  */
 const makePegasus = (zcf, board) => {
```

Comment:
IIUC, we're no longer using the multi-arg contract function style. Rather, we're writing contract functions with only the `zcf` parameter and getting the terms from the `zfc`.

Reply:
`makePegasus` creates the `publicFacet` to be used in the `start` function.